lib/std/process.zig

```diff
@@ -676,13 +676,18 @@ pub const ArgIteratorWindows = struct {
 
     /// `cmd_line_w` *must* be a WTF16-LE-encoded string.
     ///
-    /// The iterator makes a copy of `cmd_line_w` converted WTF-8 and keeps it; it does *not* take
-    /// ownership of `cmd_line_w`.
+    /// The iterator stores and uses `cmd_line_w`, so its memory must be valid for
+    /// at least as long as the returned ArgIteratorWindows.
     pub fn init(allocator: Allocator, cmd_line_w: [*:0]const u16) InitError!ArgIteratorWindows {
-        const cmd_line = try unicode.wtf16LeToWtf8Alloc(allocator, mem.sliceTo(cmd_line_w, 0));
-        errdefer allocator.free(cmd_line);
-
-        const buffer = try allocator.alloc(u8, cmd_line.len + 1);
+        const cmd_line = mem.sliceTo(cmd_line_w, 0);
```

Comment:
Not sure if it's necessarily related to this PR but thought it might be worth noting that `mem.sliceTo()` (a `while` loop) can be skipped using:
```zig
const cmd_line = std.os.windows.peb().ProcessParameters.CommandLine;
const cmd_line_slice = cmd_line.Buffer.?[0 .. cmd_line.Length / 2 :0];
```